src/jit/codegenlegacy.cpp

```diff
@@ -22444,14 +22396,6 @@ void                CodeGen::genPInvokeCallEpilog(LclVarDsc *  frameListRoot,
     {
         if (retVal == RBM_INTRET || retVal == RBM_LNGRET)
         {
-#ifdef _TARGET_AMD64_
-
-            assert(retTmp != NULL);
-            inst_RV_ST(INS_mov, REG_INTRET, retTmp, 0, TYP_LONG);
-            regTracker.rsTrackRegTrash(REG_INTRET);
-            compiler->tmpRlsTemp(retTmp);
-
-#elif defined(_TARGET_X86_)
```

Comment:
same answer.

---
In reply to: [67235291](https://github.com/dotnet/coreclr/pull/5815#discussion_r67235291) [](ancestors = 67235291)